### ISSUE TITLE
PlayGrpc code generation error

### ISSUE THREAD

**Issue by Tochemey** [playframework/play-grpc#215]
I am facing an issue while generating scala classes using PlayGrpc.
I am using:

Scala Version: 2.13.1
sbt version: 1.3.8
lagom version : 1.6

These are my settings:

```
.settings(
    inConfig(Compile)(
      Seq(
        PB.protoSources := Seq(file("account/src/main/protobuf")),
        PB.includePaths ++= Seq(file("account/src/main/protobuf")),
      )
    ),
    // Using Scala
    akkaGrpcGeneratedLanguages := Seq(AkkaGrpc.Scala),
    akkaGrpcGeneratedSources := Seq(AkkaGrpc.Server, AkkaGrpc.Client),
    akkaGrpcExtraGenerators in Compile += PlayScalaServerCodeGenerator,
    akkaGrpcCodeGeneratorSettings += "server_power_apis"
  )
```
and this is the error I am having:
```
[error] /Users/arsene/dev/accounts-service/account/target/scala-2.13/src_managed/main/com/namely/account/AbstractAccountGrpcServicePowerApiRouter.scala:11:13: object grpc is not a member of package play
[error] import play.grpc.internal.PlayRouter
[error]             ^
[error] /Users/arsene/dev/accounts-service/account/target/scala-2.13/src_managed/main/com/namely/account/AbstractAccountGrpcServicePowerApiRouter.scala:19:13: not found: type PlayRouter
[error]     extends PlayRouter(AccountGrpcService.name)
[error]             ^
[error] /Users/arsene/dev/accounts-service/account/target/scala-2.13/src_managed/main/com/namely/account/AbstractAccountGrpcServicePowerApiRouter.scala:19:43: no arguments allowed for nullary constructor Object: ()Object
[error]     extends PlayRouter(AccountGrpcService.name)
[error]                                           ^
[error] /Users/arsene/dev/accounts-service/account/target/scala-2.13/src_managed/main/com/namely/account/AbstractAccountGrpcServicePowerApiRouter.scala:5:25: Unused import
[error] import scala.concurrent.Future
[error]                         ^
[error] /Users/arsene/dev/accounts-service/account/target/scala-2.13/src_managed/main/com/namely/account/AbstractAccountGrpcServicePowerApiRouter.scala:7:35: Unused import
[error] import akka.http.scaladsl.model.{ HttpRequest, HttpResponse }
[error]                                   ^
[error] /Users/arsene/dev/accounts-service/account/target/scala-2.13/src_managed/main/com/namely/account/AbstractAccountGrpcServicePowerApiRouter.scala:7:48: Unused import
[error] import akka.http.scaladsl.model.{ HttpRequest, HttpResponse }
[error]                                                ^
[error] /Users/arsene/dev/accounts-service/account/target/scala-2.13/src_managed/main/com/namely/account/AbstractAccountGrpcServicePowerApiRouter.scala:11:27: Unused import
[error] import play.grpc.internal.PlayRouter
[error]                           ^
[error] /Users/arsene/dev/accounts-service/account/target/scala-2.13/src_managed/main/com/namely/account/AbstractRouter.scala:11:13: object grpc is not a member of package play
[error] import play.grpc.internal.PlayRouter
[error]             ^
[error] /Users/arsene/dev/accounts-service/account/target/scala-2.13/src_managed/main/com/namely/account/AbstractRouter.scala:19:13: not found: type PlayRouter
[error]     extends PlayRouter(AccountGrpcService.name)
[error]             ^
[error] /Users/arsene/dev/accounts-service/account/target/scala-2.13/src_managed/main/com/namely/account/AbstractRouter.scala:19:43: no arguments allowed for nullary constructor Object: ()Object
[error]     extends PlayRouter(AccountGrpcService.name)
[error]                                           ^
[error] /Users/arsene/dev/accounts-service/account/target/scala-2.13/src_managed/main/com/namely/account/AbstractRouter.scala:5:25: Unused import
[error] import scala.concurrent.Future
[error]                         ^
[error] /Users/arsene/dev/accounts-service/account/target/scala-2.13/src_managed/main/com/namely/account/AbstractRouter.scala:7:35: Unused import
[error] import akka.http.scaladsl.model.{ HttpRequest, HttpResponse }
[error]                                   ^
[error] /Users/arsene/dev/accounts-service/account/target/scala-2.13/src_managed/main/com/namely/account/AbstractRouter.scala:7:48: Unused import
[error] import akka.http.scaladsl.model.{ HttpRequest, HttpResponse }
[error]                                                ^
[error] /Users/arsene/dev/accounts-service/account/target/scala-2.13/src_managed/main/com/namely/account/AbstractRouter.scala:11:27: Unused import
[error] import play.grpc.internal.PlayRouter
[error]                           ^
[error] 14 errors found
[error] (account / Compile / compileIncremental) Compilation failed
```
Amazingly when I checked the classes are generated. But whenever I run `sbt compile` I got that error.

**Comment by Tochemey** [comment]
The issue I am having is that `play-grpc-runtime` is not available on scala 2.13

**Comment by raboof** [comment]
That makes sense! Would you be interested in contributing cross-building for 2.13?

**Comment by Tochemey** [comment]
@raboof I observe that the ` akka-grpc codegen` is still 2.12. I did some test locally and I got stuck

**Comment by raboof** [comment]
@Tochemey that's correct - but AFAICS that should not be a problem, since 'codegen' is only used during code generation (so 'inside' sbt, which is on scala 2.12) - you already noticed the classes are generated, it's just the runtime library that needs to be made available for 2.12.

Perhaps we should merge this issue with #40? Let's continue there.
